ups/ts_mtdome.table

```diff
@@ -4,6 +4,8 @@
 #   the "sconsUtils" package.
 setupRequired(sconsUtils)
 setupRequired(ts_salobj)
+setupRequired(ts_tcpip)
```

Comment:
I gently suggest removing scons support while you are messing around in this package (in which case remove `setupRequired(sconsUtils)` and delete files `SConstruct` and `tests/SConscript`).